src/ImageExif/ImageExif.js

```diff
@@ -0,0 +1,211 @@
+          imageLocationFeature.set('imageURL', imageURL)
+          features.push(imageLocationFeature)
+
+          // Image Location Error Margin
+          const imageLocationError = Number(sortedProperties.GPSHPositioningError)
+
+          if (imageLocationError) {
+            const errorRadius = new Circle(coords, imageLocationError)
+            const errorRadiusFeature = new Feature(errorRadius)
+
+            errorRadiusFeature.setProperties({ 'Error Radius (meters)': imageLocationError })
+            errorRadiusFeature.set('title', 'Image Location Error Radius')
+            errorRadiusFeature.set('name', 'Error Radius')
+            errorRadiusFeature.set('imageId', imageId)
+            errorRadiusFeature.set('imageURL', imageURL)
+            errorRadiusFeature.setStyle(new Style({
+              stroke: new Stroke({
+                lineDash: [4, 8],
+                color: '#ff0000',
+                width: 2
+              })
+            }))
+            features.push(errorRadiusFeature)
+          }
+
+          // Image Bearing Line
+          const imageBearing = sortedProperties.GPSDestBearing
+
+          if (imageBearing) {
+            const geojsonPoint1 = olToGeojson(imageLocationFeature)
+            const geojsonPoint2 = transformTranslate(geojsonPoint1, 2, imageBearing, { units: 'meters' })
+            const olPoint1 = geojsonToOl(geojsonPoint1).getGeometry().getCoordinates()
+            const olPoint2 = geojsonToOl(geojsonPoint2).getGeometry().getCoordinates()
+            const lineThing = new LineString([olPoint1, olPoint2])
+            const bearingFeature = new Feature(lineThing)
+
+            bearingFeature.set('title', 'Image Bearing')
+            bearingFeature.set('name', 'Image Bearing')
+            bearingFeature.set('imageId', imageId)
+            bearingFeature.setStyle(styleFunction)
+            bearingFeature.set('imageURL', imageURL)
+            features.push(bearingFeature)
+          }
+
+          const source = new VectorSource({ features })
+          const vectorLayer = new VectorLayer({ source })
+
+          vectorLayer.set('title', 'Image Location')
+
+          map.addLayer(vectorLayer)
+        } else {
+          unlocatableFiles.push(file)
+        }
+      }
+      reader.onerror = function (e) {
+        // eslint-disable-next-line no-console
+        console.log(e.target.error)
+      }
+      reader.readAsArrayBuffer(file)
+    })
+  }, [])
+  const { getRootProps, getInputProps, isDragActive } = useDropzone({ accept: 'image/*', onDrop })
+
+  return (
+    <Container>
+      <Title>{translations['_ol_kit.ImageExif.title']}</Title>
+      <InnerContainer>
+        <InputContainer>
+          <div {...getRootProps()}>
+            <input {...getInputProps()} />
+            {
+              isDragActive
+                ? <p>Drop the files here ...</p>
+                : <p>Drag and drop some image files here, or click to select files</p>
```

Comment:
can you add these to translations, thanks

Reply:
Fixed!